### DISCUSSION TITLE
Key-mapper updating

### DISCUSSION THREAD

**Discussion by a-e-egorov** [sezanzeb/input-remapper#190] [Q&A]
Is there any way to update Key-mapper kind of "on the fly" - i.e. without erasing the old version and putting the new one, and without loosing all the previous settings?

**Comment by sezanzeb** [comment]
Settings in `~/.config/key-mapper` should not be lost when installing a new version, I have never observed that.

Is that file gone when you install a new version?

**Reply by a-e-egorov** [reply]
I tried to update now - in fact it went well, setting were preserved.
